docs/pages/starter-kit/configuration.mdx

```diff
@@ -7,27 +7,19 @@ layout: 'full'
-| NEXT_PUBLIC_CHAIN_ID                    | Chain ID used for the application                                       | 1                                         |
-| NEXT_PUBLIC_REPORT_EMAIL                | Email to use for report                                                 | report@acme.com                           |
-| NEXT_PUBLIC_HOME_TOKENS                 | List of tokens to display in the featured tokens (separated by a comma) | 1-0x...-123...,1-0x...-456,1-0x...789     |
-| NEXT_PUBLIC_BUGSNAG_API_KEY             | Key for Bugsnag to track errors                                         | xxxxxx                                    |
-| NEXT_PUBLIC_OFFER_VALIDITY_IN_SECONDS   | Duration in second of a valid offer                                     | 2419200                                   |
-| NEXT_PUBLIC_AUCTION_VALIDITY_IN_SECONDS | Duration in second of a valid bid for an auction                        | 604800                                    |
-| NEXT_PUBLIC_BASE_URL                    | Base URL for your platform                                              | https://marketplace.acme.com              |
-| NEXT_PUBLIC_GA_MEASUREMENT_ID           | Google analytics ID                                                     | G-XXX                                     |
-| NEXT_PUBLIC_MINTABLE_COLLECTIONS        | List of collection addresses available for mint (separated by a comma)  | 0x123...,0x456...                         |
-| EMAIL_HOST                              | Host configuration for your email provider                              | smtp.sendgrid.net                         |
-| EMAIL_PORT                              | Port configuration for your email provider                              | 587                                       |
-| EMAIL_USERNAME                          | Username configuration for your email provider                          | apikey                                    |
-| EMAIL_PASSWORD                          | Password configuration for your email provider                          | SG.xxx                                    |
-| EMAIL_FROM                              | Email from configuration for your email provider                        | Liteflow \<contact\@liteflow.com\>        |
-| LITEFLOW_WEBHOOK_SECRET                 | Webhook secret to receive event from the Liteflow infrastructure        | xxxxxx                                    |
+| Variable                              | Description                                                      | Value example                      |
+| ------------------------------------- | ---------------------------------------------------------------- | ---------------------------------- |
+| NEXT_PUBLIC_LITEFLOW_BASE_URL         | Liteflow base URL for your platform                              | https://api.liteflow.com           |
```

Comment:
Keep in mind this one is optional and will be required only for us to use the staging so I would recommend not to document it
```suggestion
```

Reply:
[Remove NEXT_PUBLIC_LITEFLOW_BASE_URL from configuration](https://github.com/liteflow-labs/liteflow-js/pull/163/commits/88c87c915fc19948545428160c895592f43e645f)